tests/StringyTest.php

```diff
@@ -546,6 +546,19 @@ public function testTruncate($expected, $str, $length, $substring = '',
     }
 
     /**
+     * @dataProvider safeTruncateProvider()
+     */
+    public function testSafeTruncate($expected, $str, $length, $substring = '',
+                                     $encoding = null)
```

Comment:
I think 4.4 of psr-2 is one of the only things we hadn't been following.

The difference:

``` php
public function testSafeTruncate($expected, $str, $length, $substring = '',
                                 $encoding = null)
{
    // body
}

public function testSafeTruncate(
    $expected,
    $str,
    $length,
    $substring = '',
    $encoding = null
) {
    // body
}
```